core/FrontController.php

```diff
@@ -428,8 +428,13 @@ public function init()
         if (Common::getRequestVar('token_auth', '', 'string') !== ''
             && Request::shouldReloadAuthUsingTokenAuth(null)
         ) {
+            if (!Common::isPhpCliMode()) {
```

Comment:
@flamisz might be worth trying not to use the session authenticator when a token_auth is used in the query. Eg return `null` in `makeSessionAuthenticator()` if a session is set? Unsetting super user access etc may have side effects not sure